release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -559,6 +559,15 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 
 The Special Resource Operator (SRO) now exposes metrics to help you watch the health of your SRO custom resources and objects. For more information, see xref:../hardware_enablement/psap-special-resource-operator.html#special-resource-operator-metrics_special-resource-operator[Prometheus Special Resource Operator metrics].
 
+[id="ocp-4-10-nto-metrics"]
+==== New Node Tuning Operator metric added to Telemetry
+
+A Node Tuning Operator (NTO) metric is now added to Telemetry. Follow xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Showing data collected by Telemetry] to see all the metrics collected by Telemetry.
```

Comment:
```suggestion
A Node Tuning Operator (NTO) metric is now added to Telemetry. xref:../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Showing data collected by Telemetry] details all of the metrics collected by Telemetry.
```
Purely a suggestion to avoid `Follow`.